### DISCUSSION TITLE
Potential Bug

### DISCUSSION THREAD

**Discussion by Genjidegaulle** [beetbox/beets#4844] [Q&A] [answered]
Hi, 

First post here so I'm sharing in the discussion before raising an issue in GitHub.

I'm looking to tag a release album that's been uploaded onto MusicBrainz [here](https://musicbrainz.org/release-group/70c86866-ece7-4576-adcd-4a059df765a0).

However, beet can't seem to find the album even when I enter the id. Here are the logs when I run verbose:
![image](https://github.com/beetbox/beets/assets/22668252/68a0b319-b4fa-4559-a500-582e126c2723)

Specifically, it seems like no tracks are found, even though there exists one in the album. 

Let me know if I should provide more information!

**Comment by sampsyo** [comment]
Hi! You've linked to a "release group," which is MusicBrainz's term for a group of different versions of the same album. You will need to instead paste the ID of a specific release. You can actually paste the whole URL if you want:
https://musicbrainz.org/release/d2718a7f-8168-4b3e-b108-3511840b7a81

**Reply by Genjidegaulle** [reply]
Hey, thanks for the help! I embarrassingly posted the wrong link in my first post, but my screenshot above shows that I did paste the actual release ID (instead of just the group). Here's the output when I put the whole URL: 
![image](https://github.com/beetbox/beets/assets/22668252/a9a93df8-7c1f-4f33-9264-494d7648e662)

What's strange is that there seem to be 0 tracks found by the api even though there's clearly one track in the single:
![image](https://github.com/beetbox/beets/assets/22668252/7fb91941-071d-4d4f-ac88-69665991148e)

I'll try to look into the code but would appreciate some pointers as I haven't contributed before.

**Reply by sampsyo** [reply, marked as answer]
Ah—that's a video track. The importer ignores video tracks by default. Fortunately, that's configurable:
https://beets.readthedocs.io/en/stable/reference/config.html#ignore-video-tracks

**Reply by Genjidegaulle** [reply]
This is my config file after changes: 
![image](https://github.com/beetbox/beets/assets/22668252/0aabc142-9e6d-4540-a951-ec5f224ed813)

and it worked out! Thanks so much for the help!
